wiki/Medals/en.md

```diff
@@ -1,59 +1,295 @@
+| Spooked | Something moved. It wasn't your cursor! *(Don't look behind you.)* |
+| Skylord | Never miss a wingbeat. *(Flight among the sky requires nothing but perfection.)* |
+| B-Rave | It takes courage to stand before the master. *(Ill-tempered bosses make for excellent final battles.)* |
+| Any\% | A speedrunner's best friend. *(Sometimes you have to break the rules to work with speed.)* |
+| Mirage | The horizon goes forever, and ever, and ever.. *(The light is far harder than it seems.)* |
+| Under The Stars | Onwards, to where the darkness can never stop us. *(Walk beneath the stars on a journey to elsewhere.)* |
+| Senseless | I see nothing. I hear nothing. *(A song that is, and something that is not.)* |
+| Aeon | In the mire of thawing time, memory shall be your guide. *(When time runs slow and sight fails you, how will you succeed?)* |
+| Upon The Wind | And in that gale, no eye could hope to follow. *(In the heart of a storm of flowers, a world unseen.)* |
+| Vantage | There we stood, where the spires pierced the sky, and dreamed of the future to come. *(Where ground was broken to critical acclaim.)* |
+| Quick Maths | Beats per minute over.. this isn't quick at all! *(Where x equals beats per minute, and a variable unknown..)* |
+| Efflorescence | A lament for the past, and a glimpse into tomorrow. *(The horizon thunders with a portentous bloom.)* |
+| Inundate | Swept away. *(Across and beyond the tides.)* |
+| Not Bluffing | Did that with my eyes closed. *(It's not arrogance if you can live up to it.)* |
+| Eureka | By Jove, you got it! *(When inspiration strikes..)* |
+| Regicide | A king no more. *(No throne lasts forever.)* |
+| Permadeath | One life, one shot. *(Better make it count! Ware well the example of the white cat.)* |
+| The Future Is Now | The stars showed you all that was to come. *(Withstand the primordial.)* |
+| Natural 20 | Rolled it. *(Not just a chance of the dice, this one is all skill.)* |
+| Kaleidoscope | So many pretty colors. Most of them red. *(See the tiny colours up through the tube up close and personal, and slow right down.)* |
+| AHAHAHAHA | Together forever. *(Who am I? Where am I? What the hell IS this place?)* |
+| Valediction | One last time. *(Time stood still as we waved farewell.)* |
+
+## Beatmap Spotlights
+
+The **Beatmap Spotlights medals** are medals that are being awarded upon the completion of certain beatmap packs that are being released as a part of the [Monthly Beatmap Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights). In order to unlock the medals in this category, a player needs to play through all the maps *in any of the four gamemode packs* that is being contested on the corresponding Beatmap Spotlights ranking chart. The usage of either [difficulty reduction mods](/wiki/Game_modifier#difficulty-reduction) or [difficulty increase mods](/wiki/Game_modifier#difficulty-increase) is permitted.
```

Comment:
```suggestion
*Beatmap Spotlights medals* are medals that are awarded upon the completion of certain beatmap packs that are being released as a part of the [Monthly Beatmap Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights). In order to unlock the medals in this category, a player needs to play through all the maps in any of the four gamemode packs that are contested on the corresponding Beatmap Spotlights ranking chart; the usage of either [difficulty reduction mods](/wiki/Game_modifier#difficulty-reduction) or [difficulty increase mods](/wiki/Game_modifier#difficulty-increase) is permitted.
```

Reply:
think i'll keep the italic and the full stop here but changed the rest